UI/ClientUI.cpp

```diff
@@ -476,101 +476,100 @@ namespace {
+        db.Add<std::string>("audio.music.path",                 UserStringNop("OPTIONS_DB_BG_MUSIC"),                          (GetRootDataDir() / "default" / "data" / "sound" / "artificial_intelligence_v3.ogg").string());
+        db.Add("audio.music.volume",                            UserStringNop("OPTIONS_DB_MUSIC_VOLUME"),                      127,                    RangedValidator<int>(1, 255));
+        db.Add("audio.effects.volume",                          UserStringNop("OPTIONS_DB_UI_SOUND_VOLUME"),                   255,                    RangedValidator<int>(0, 255));
+        db.Add<std::string>("ui.control.button.rollover.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_BUTTON_ROLLOVER"),     (GetRootDataDir() / "default" / "data" / "sound" / "button_rollover.ogg").string());
+        db.Add<std::string>("ui.control.button.click.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_BUTTON_CLICK"),           (GetRootDataDir() / "default" / "data" / "sound" / "button_click.ogg").string());
+        db.Add<std::string>("ui.control.button.turn.click.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_TURN_BUTTON_CLICK"), (GetRootDataDir() / "default" / "data" / "sound" / "turn_button_click.ogg").string());
+        db.Add<std::string>("ui.control.listbox.select.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_LIST_SELECT"),          (GetRootDataDir() / "default" / "data" / "sound" / "list_select.ogg").string());
+        db.Add<std::string>("ui.control.listbox.drop.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_ITEM_DROP"),              (GetRootDataDir() / "default" / "data" / "sound" / "list_select.ogg").string());//TODO: replace with dedicated 'item_drop' sound
+        db.Add<std::string>("ui.control.dropdownlist.click.sound.path", UserStringNop("OPTIONS_DB_UI_SOUND_LIST_PULLDOWN"),    (GetRootDataDir() / "default" / "data" / "sound" / "list_pulldown.ogg").string());
+        db.Add<std::string>("ui.input.keyboard.sound.path",     UserStringNop("OPTIONS_DB_UI_SOUND_TEXT_TYPING"),              (GetRootDataDir() / "default" / "data" / "sound" / "text_typing.ogg").string());
+        db.Add<std::string>("ui.window.minimize.sound.path",    UserStringNop("OPTIONS_DB_UI_SOUND_WINDOW_MAXIMIZE"),          (GetRootDataDir() / "default" / "data" / "sound" / "window_maximize.ogg").string());
+        db.Add<std::string>("ui.window.maximize.sound.path",    UserStringNop("OPTIONS_DB_UI_SOUND_WINDOW_MINIMIZE"),          (GetRootDataDir() / "default" / "data" / "sound" / "window_minimize.ogg").string());
+        db.Add<std::string>("ui.window.close.sound.path",       UserStringNop("OPTIONS_DB_UI_SOUND_WINDOW_CLOSE"),             (GetRootDataDir() / "default" / "data" / "sound" / "window_close.ogg").string());
+        db.Add<std::string>("ui.messagebox.sound.path",  UserStringNop("OPTIONS_DB_UI_SOUND_ALERT"),                    (GetRootDataDir() / "default" / "data" / "sound" / "alert.ogg").string());
```

Comment:
`messagebox` isn't obviously related to alert popups... maybe could be clearer...

Reply:
`ui.dialog.alert.sound.path`?

Reply:
lose the `dialog` ?